de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/ui/model/roster/RosterAdapterFactory.java

```diff
@@ -13,8 +13,8 @@
  * <p>E.g. let's you adapt {@link RosterGroupElement} to {@link RosterGroup} with
  *
  * <pre>
- * RosterGroup rosterGroup = (RosterGroup) rosterGroupElement
- *     .getAdapter(RosterGroup.class);
+ * RosterGroup rosterGroup = rosterGroupElement.adaptTo(RosterGroup.class);
```

Comment:
Nope, this the ISelectionService interface

Reply:
My bad. Forgot that the refactoring logic can't differentiate methods in plaintext comments. Will pay more attention in the future.
Fixed.